conversant/demo/ui.py

```diff
@@ -76,7 +76,7 @@ def draw_chat_history() -> None:
                 )
             if "bot" in turn:
                 st_message(
-                    value=turn["bot"].rstrip(),
+                    value=turn["bot"],
```

Comment:
I am guessing the changes in this file address the bug where the output is printed twice. Would you mind providing some intuition as to why removing `lstrip`/`rstrip` this fixes the issue? Or is it related to the changes in `styles.css`?

Reply:
Because of how we determine where to animate from ([ref](https://github.com/yichern/st-chat/blob/60ca345d309961a2f6b134bb60bb988f0d048a45/streamlit_talk/frontend/src/stChat.tsx#L43)), if

```
value = "test"
animateFrom = "test\n"
```
then 
`value.split(animateFrom).join('')` evaluates to `test`, causing a repeat of the last utterance. Removing `.rstrip()` allows value to be `test\n`, so `value.split(animateFrom).join('')` evaluates to `""`